design_linked_list.py

```diff
@@ -0,0 +1,73 @@
+class Node:
+    def __init__(self, val = 0, next = None):
+        self.val = val
+        self.next = next
+        
+class MyLinkedList:
+
+    def __init__(self):
+        self.head = Node()
+        self.length = 0
+
+    def get(self, index: int) -> int:
+        if index >= self.length:
+            return -1
+        
+        cur = self.head.next
+        for _ in range(index):
+            cur = cur.next
+        
+        return cur.val
+        
+
+    def addAtHead(self, val: int) -> None:
+        cur = self.head.next
+        newNode = Node(val, cur)
+        self.head.next = newNode
+        self.length += 1
+
```

Comment:
even tho its used by many programmers and seen as a good variable naming, **cur** might be a be a little confusing sometimes